tests/test.rs

```diff
@@ -710,11 +710,13 @@ fn test_plaintext_pass_wrong_pass() {
 }
 
 #[test]
+#[cfg_attr(not(feature = "rust-crypto"), ignore)]
```

Comment:
`#[cfg(feature = "rust-crypto")]` will exclude the test entirely unless this feature is enabled.